problemsets-mat223/linearalbegra.tex

```diff
@@ -2342,37 +2342,67 @@ \section*{Subspaces and Bases}
 						Then $\alpha\vec u=\mat{0\\\alpha t}\in C$. 
 				\end{enumerate}
 			\end{solution}
-		\item $D=\{\vec x\in\R^2:\vec x=\mat{0\\t}+\mat{1\\1}\text{ for some }t\in\R\}$.
+		\item $D=\Set*{\vec x\in\R^2 \given \vec x=\mat{0\\t}+\mat{1\\1}\text{ for some }t\in\R}$.
 			\begin{solution}
 				XXXX
 					
 				$D$ is not a subspace, since for example $\mat{1\\1}\in D$, but
 				$0\mat{1\\1}=\mat{0\\0}\notin D$. 
 			\end{solution}
-		\item $E=\{\vec x\in\R^2:\vec x=\mat{0\\t}\text{ or }\vec x=\mat{t\\0}\text{ for some }t\in\R\}$.
+		\item $E=\Set*{\vec x\in\R^2 \given \vec x=\mat{0\\t}\text{ or }\vec x=\mat{t\\0}\text{ for some }t\in\R}$.
 			\begin{solution}
 				XXXX
 
 				$E$ is not a subspace, since for example  $\mat{1\\0}$ and $\mat{0\\1}$
 				are both in $E$, but their sum is $\mat{1\\1}$ which is not in $E$.
 			\end{solution}
-		\item $F=\{\vec x\in\R^2:\vec x=t\mat{3\\1}\text{ for some }t\in\R\}$.
+		\item $F=\Set*{\vec x\in\R^2 \given \vec x=t\mat{3\\1}\text{ for some }t\in\R}$.
 			\begin{solution}
 				XXXX
 					
 				$F$ is a subspace. 
+
+				\begin{enumerate}[label=(\roman*)]
+					\item Let $\vec u,\vec v\in F$. Then $\vec u=t\mat{3\\1}$ and
+						$\vec v=s\mat{3\\1}$ for some $s,t\in\R$. 
+
+						But then $\vec u+\vec v=(s+t)\mat{3\\1}\in F$. 
+					\item Let $\vec u=t\mat{3\\1}\in F$ and let $\alpha$ be any scalar.
+						Then $\alpha\vec u=(\alpha t)\mat{3\\1}\in F$. 
+				\end{enumerate}
 			\end{solution}
 		\item $G=\Span\left\{\mat{1\\1}\right\}$.
 			\begin{solution}
 				XXXX
 					
 				$G$ is a subspace. 
+
+				By definition of a span,
+				$G=\Set*{\vec x\in\R^2 \given \vec x=t\mat{1\\1}\text{ for some }t\in\R}$.
+
+				The proof that $G$ is a subspace is essentially the same as the
+				proof for $F$ above.
```

Comment:
Let me know if you want me to write this one out in full as well.

Reply:
Yes. You see that it's the same, but students do not.

Reply:
No problem. Mostly a copy/paste!